testscript/testdata/testscript_notfound.txt

```diff
@@ -0,0 +1,9 @@
+# Check that unknown commands output the right error
```

Comment:
what about joining all three scripts into one, with sections for each of the three cases? a total of ~30 lines doesn't feel like enough to warrant splitting into multiple files. you can name the inner scripts like `-- scripts/notfound_misspelled.txt --`.

Reply:
I created multiple files because I couldn't figure out how to tell it what output to expect for which script. Not terribly fluent in `testscript` :) How about this?

Reply:
SGTM!